dev/Deployment/DeploymentManager.cpp

```diff
@@ -327,29 +337,38 @@ namespace winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::implem
     HRESULT DeploymentManager::VerifyPackage(const std::wstring& packageFamilyName, const PACKAGE_VERSION targetVersion,
         __out std::wstring& packageIdentifier) try
     {
-        auto packageFullNames{ FindPackagesByFamily(packageFamilyName) };
+        winrt::Windows::Management::Deployment::PackageManager packageManager;
+        auto packages{ packageManager.FindPackagesForUser(L"", packageFamilyName) };
```

Comment:
You want all packages in the family? Even Bundle? Resource? Optional?

`FindPackagesForUserWithPackageType(...Main|Framework)` ?

Reply:
Should have checked FindPackagesByFamily's definition for proper like for like winrt API replacement and should also have thought beyond like for like replacement here. That also points that FindPackagesByFamily is half complete in it's name as it is filtering in it's find for ONLY Main and Framework package types only and not all package types. 

Anyway, figured DeploymentManager::FindPackagesByFamily is not used and also there is a duplicate API, FindByFamily, in dev\common\AppModel.Package.h that does the same.

Probably, the FindByFamily in dev\common\AppModel.Package.h should be renamed as FindMainOrFrameworkPackagesByFamily, to be more precise in it's name ?